.github/workflows/windows-package.yml

```diff
@@ -49,6 +41,11 @@ jobs:
           ls -l
           python setup.py develop
           py.test -v
-    - run:
+    - run: |
           python setup.py bdist_wheel
       if: ${{ github.event_name == 'release' }}
+           python -m pip install twine
```

Comment:
In this location as well, the `if` statement must be outside the `run` statement.

Reply:
please check I write if condition above run statement